### PR TITLE
add v, v3, v8 integ datasets

### PR DESCRIPTION
**Pull request by RoyStegeman**

### CODE REVIEW

File: nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    mid: 1.0e-09
+    max: null
+  k2:
+    min: null
+    mid: 2.7225
+    max: null
+  k3:
+    min: null
+    mid: 0.0
+    max: null
```
Comment (scarlehoff) [inline nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:13]: I think they should all read from the same kinematics file given that's the same.

Also, add the right variables :__

File: nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+bins:
+- k1:
```
Comment (scarlehoff) [inline nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:2]:
```suggestion
- x:
```

File: nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    min: null
+    mid: 1.0e-09
+    max: null
+  k2:
```
Comment (scarlehoff) [inline nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:6]:
```suggestion
  Q:
```

File: nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/metadata.yaml [hunk 4]
```diff
@@ -92,3 +92,123 @@ implemented_observables:
...
+  ndata: 1
+  plotting:
+    kinematics_override: dis_sqrt_scale
+    theory_reference: Bertone:2013vaa
+    dataset_label: 'integrability dataset: $xV$ PDF, Q=Q_0'
+    plot_x: k1
+  kinematic_coverage:
+  - k1
+  - k2
+  - k3
```
Comment (scarlehoff) [inline nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/metadata.yaml:112]:
```suggestion
  - x
  - Q
```

You should be able to get away with defining only two.

File: nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml (hunk 1 above)
Comment (RoyStegeman) [reply nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:13]: To both points: I just followed the implementation of the already existing ones, but I suppose I can fix all

Comment (scarlehoff) [reply nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:13]: If you fix them all even better.

Btw, if you change the processes from `DIS_whatever` to `DIS_INTEG_whatever` I'd appreciate it.

Comment (RoyStegeman) [reply nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:13]: https://github.com/NNPDF/nnpdf/pull/2059/commits/928a27f5f80e7ae4c96f298823606e4bf5434886

Just `INTEG_` I guess, or do you want to ask @comane to change it?

Comment (scarlehoff) [reply nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/kinematics_XV8.yaml:13]: Ah, yes. Even better.

**Comment by RoyStegeman** [comment]
Why is Bertone:2013vaa the theory reference for integrability?

**Comment by scarlehoff** [comment]
Due to this file https://github.com/NNPDF/nnpdf/blob/master/nnpdf_data/nnpdf_data/commondata/PLOTTINGTYPE_DIS.yaml  

(the conversion script just took vp information as the truth, incidentally, all DIS datasets are wrong because the concept of theory reference probably doesn't make any sense since it can come from different sources)

**Comment by RoyStegeman** [comment]
But INTEG is just the PDF, not DIS

**Comment by scarlehoff** [comment]
But the process type was set to DIS. Probably as a shorthand for "just one PDF involved"

**Review by scarlehoff — APPROVED** [review]

**Pull request merged**